doc/developers-guide/mod_muc_light_developers_guide.md

```diff
@@ -71,18 +71,18 @@ There are 2 test suites and one helper module in `test/ejabberd_tests/tests`.
   `muc_light_SUITE.erl` equivalent that uses XEP-0045 compatibility mode.
 
 * `muc_helper.erl`
-  Provides handy iterators over room participants. 
+  Provides handy iterators over room participants.
   Used in MUC Light suites but in the future could be used in `muc_SUITE` as well.
 
 Hooks handled by this extension
 -----------------------
 
 * `offline_groupchat_message_hook` handled by `mod_muc_light:prevent_service_unavailable/3` - Prevents the default behaviour of sending `service-unavailable` error to the room when a groupchat message is sent to an offline occupant.
 
-* `remove_user` handled by `mod_muc_light:remove_user/2` - Triggers DB cleanup of all data related to the removed user. 
+* `remove_user` handled by `mod_muc_light:remove_user/2` - Triggers DB cleanup of all data related to the removed user.
  Includes a broadcast of a notification about user removal from occupied rooms.
 
-* `disco_local_items` handled by `mod_muc_light:get_muc_service/5` - Adds MUC service item to the Disco result. 
+* `disco_local_items` handled by `mod_muc_light:get_muc_service/5` - Adds MUC service item to the Disco result.
```

Comment:
`disco_local_items` handled by `mod_muc_light:get_muc_service/5` - Adds a MUC service item to the Disco result.